package.json

```diff
@@ -49,7 +49,7 @@
     "watch:js:switcher": "watchify utils/switcher/switcher.js -v -t babelify -g browserify-shim -o dist/switcher.js",
     "watch:test:browserify": "watchify test/browserify-test.js -o dist-test/browserify-test.js",
     "watch:test:webpack": "watchify test/webpack-test.js -o dist-test/webpack-test.js",
-    "watch:test": "npm-run-all -p watch:test:js watch:test:webpack watch:test:browserify",
+    "watch:test": "npm-run-all -p watch:test:manifest watch:test:js watch:test:webpack watch:test:browserify",
     "watch:test:js": "node scripts/watch-test.js",
     "watch:test:manifest": "node -e \"var b=require('./scripts/manifest-data.js'); b.watch();\"",
     "prepublish": "npm run build"
```

Comment:
Might want to use [in-publish](https://www.npmjs.com/package/in-publish) so this doesn't run on `npm install`